include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -379,6 +379,16 @@ __get_host_access(_Buf&& __buf)
 #endif
 }
 
+template <typename _Acc>
+auto __get_accessor_ptr(const _Acc& __acc)
```

Comment:
I would've said that the `const` here is unnecessary but I looked up in the spec that `get_pointer` is a const method (that internally `const_cast`s away the constness) so this seems fine to me.